account/models.py

```diff
@@ -46,21 +50,21 @@ class CustomUser(AbstractBaseUser, PermissionsMixin):
     
     gender = models.CharField(max_length=1, choices=GENDER)
 
-    phone_prefix = models.ForeignKey(PhonePrefix, on_delete=models.CASCADE)
+    phone_prefix = models.ForeignKey(PhonePrefix, on_delete=models.CASCADE, null=True)
     phone = models.CharField(max_length=7, validators=[validate_phone])
     SERIES = [
-        ('1', 'AZE'),
-        ('2', 'AA'),
-        ('3', 'MYI'),
-        ('4', 'DYI')
+        ('AZE', 'AZE'),
```

Comment:
You might want to check TextChoices class

Reply:
I checked TextChoices. But since choice values are short, I didnt see a need to use it in this scenario.